test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -823,12 +823,49 @@ TEST_F(TransferAssetTest, InvalidWhenWrongPrecision) {
   ASSERT_FALSE(validateAndExecute());
 }
 
-TEST_F(TransferAssetTest, InvalidWhenDifferentCreator) {
+TEST_F(TransferAssetTest, InvalidWhenCreatorHasNoPermission) {
```

Comment:
For some reason I have different output from clang-format (following as well):
```
TEST_F(TransferAssetTest, InvalidWhenCreatorHasNoPermission) {
  // Transfer creator is not connected to account
  transfer_asset->src_account_id = account_id;
  transfer_asset->dest_account_id = admin_id;
  EXPECT_CALL(*wsv_query,
              hasAccountGrantablePermission(admin_id, account_id, can_transfer))
      .WillOnce(Return(false));
  ASSERT_FALSE(validateAndExecute());
}
```
Did you use it?